chalicelib/checks/helpers/qc_utils.py

```diff
@@ -1,6 +1,37 @@
 from dcicutils import ff_utils
 
 
+def calculate_qc_metric_rnaseq(file_uuid, key):
```

Comment:
As an aside, I'm noticing that none of the check related functions in `helpers` are directly tested. It may be worthwhile at some point to test these in some way. This is probably a task for when we split Foursight into two repos.

Reply:
Sounds good.